phplib/admin-reps.php

```diff
@@ -85,7 +85,7 @@ private function renderReps($self_link, $reps, $bad_link = false)
             if ($bad_link && $i < count($reps) - 1) {
                 $link_extra = "&nextbad=".urlencode($reps[$i+1]);
             }
-            $html .= "<a href=\"$self_link&pc=" . urlencode(get_http_var('pc')) . "&rep_id=" . $rep
+            $html .= "<a href=\"$this->self_link&pc=" . urlencode($this->params['pc']) . "&rep_id=" . $rep
```

Comment:
It comes from commonlib's phplib/admin.php which sets it before then calling `$page->display`.